aiinfra-cluster/modules/dashboard/outputs.tf

```diff
@@ -0,0 +1,446 @@
+/**
+  * Copyright 2022 Google LLC
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+// reference:
+// https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/tree/master/dashboards/nvidia-gpu
```

Comment:
Do we need to or want to keep these in sync?

Reply:
There is no need to keep these 2 in sync IMO. The dashboard from the provisioning tool can be developed independently based on the requirements. But any new telemetry we definitely need to add to the provisioning tool dashboard.